client.lua

```diff
@@ -897,7 +897,12 @@ exports("GetPlayer", function(label) return Players[label] end)
 
 exports("UpdateType", function(type, label, data) Types[type][label] = data end)
 
-exports("UpdateZone", function(name, data) Zones[name] = data end)
+local function UpdateZoneOptions (name, targetoptions)
+	targetoptions.distance = targetoptions.distance or Config.MaxDistance
+	Zones[name].targetoptions = targetoptions
+end
+
+exports("UpdateZoneOptions", function(name, targetoptions) UpdateZoneOptions(name, targetoptions) end)
```

Comment:
```suggestion
exports("UpdateZoneOptions", UpdateZoneOptions)
```
This is more minimalistic written, works the exact same as before

Reply:
Yep u update just the targetting options and not the complete zone ;)